glossary.qmd

```diff
@@ -0,0 +1,353 @@
+- `Unsigned short` or `Uint32`: unsigned integer with 32 bit capacity, which can hold values from 0 to 4,294,967,295 (inclusive).
+- `Long` or `Int64`: signed integer with 64 bit capacity, which can hold values from -9223372036854775808 to 9223372036854775807 (inclusive).
+- `Unsigned long` or `Uint64`: unsigned integer with 64 bit capacity, which can hold values from 0 to 18446744073709551615 (inclusive).
+- `float`: 32 bit floating point number.
+- `double`: 64 bit floating point number.
+
+For a good explainer on how floating point numbers work, refer to [this blog post](https://ciechanow.ski/exposing-floating-point/).
+
+#### Deflate
+
+A [lossless](#lossless-compression) [compression](#compression) codec used as part of [ZIP archives](#zip-archive) and internally within [COG](#cloud-optimized-geotiff-cog) and [GeoTIFF](#geotiff) files.
+
+#### Entwine Point Cloud (EPT)
+
+A [cloud-optimized](#cloud-optimized) file format for [point cloud data](#point-cloud-data). Entwine has largely been superseded by [COPC](#cloud-optimized-point-cloud-copc) because COPC is backwards-compatible with previous point cloud data formats.
+
+#### EPSG code
+
+A [projection definition](#coordinate-reference-system-crs) referring to the EPSG database. EPSG codes tend to be four or five digits and tend to be easier to remember and use than longer definitions, such as [WKT strings](#wkt-projection-definition) or [PROJJSON](#projjson). The downside of EPSG codes is that the program needs to have the EPSG database available so that it can perform a lookup from the EPSG code to the full projection definition.
+
+#### External compression
+
+[Compression](#compression) that is not part of a file format's own specification, and which is added on after the main file has been saved. This tends to be used as part of [ZIP archives](#zip-archive) (with file extension `.zip`) or with standalone [gzip compression](#gzip) (file extension `.gz`).
+
+External compression tends to make a file no longer [cloud-optimized](#cloud-optimized), as it is usually no longer possible to read part of the file without fetching the entire file, as the entire file is necessary for decompression.
+
+This is in contrast to [internal compression](#internal-compression).
+
+#### fsspec
+
+A Python library for abstracting among several different [cloud storage](#object-storage-cloud-storage) providers. Allows uploading and downloading files to each provider with a consistent API.
+
+#### GDAL
+
+The _Geospatial Data Abstraction Library_, a widely-used open-source library for converting between different [raster data](#raster-data) formats, as well as reprojecting between [coordinate reference systems](#coordinate-reference-system-crs).
+
+Its common command-line tools include `gdalinfo` and `gdal_translate`. It can be used from Python with the `rasterio` library or from R with the `terra` library.
+
+#### GeoDataFrame
```

Comment:
Is this the Python specific terminology? sf in R is equivalent

Maybe abstract to Geospatial Data Frame and not in geospandas and sf what these are called?